src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -823,6 +832,28 @@ private void WriteProjectItems(IEnumerable items)
             }
         }
 
+        private const string EmbedInBinlogItemType = "EmbedInBinlog";
```

Comment:
Put special item types known by the engine here: [https://github.com/dotnet/msbuild/blob/main/src/Shared/Constants.cs#L126](https://github.com/dotnet/msbuild/blob/main/src/Shared/Constants.cs#L126)

Reply:
good idea, done